gifted_build/postgres_manage.js

```diff
@@ -0,0 +1,97 @@
+const { Client } = require('pg');
+const knex = require('knex')({
```

Comment:
this connection is fine for the testing environment and for development purposes, but you should rename the file as this is really only meant for development purposes and setting up the production environment should be its own file. I realllly don't want any drop table commands to be near production.